apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -49,17 +49,22 @@ const (
 	StatusAnnotation = "baremetalhost.metal3.io/status"
 
 	// RebootAnnotationPrefix is the annotation which tells the host which mode to use
-	// when rebooting - hard/soft
+	// when rebooting - hard/soft.
 	RebootAnnotationPrefix = "reboot.metal3.io"
 
 	// InspectAnnotationPrefix is used to specify if automatic introspection carried out
-	// during registration of BMH is enabled or disabled
+	// during registration of BMH is enabled or disabled.
 	InspectAnnotationPrefix = "inspect.metal3.io"
 
 	// HardwareDetailsAnnotation provides the hardware details for the host
 	// in case its not already part of the host status and when introspection
-	// is disabed
+	// is disabled.
 	HardwareDetailsAnnotation = InspectAnnotationPrefix + "/hardwaredetails"
+
+	// Disabled is a constant string="disabled"
+	// This is particularly useful to check if inspect annotation is disabled
+	// inspect.metal3.io=disabled.
+	Disabled = "disabled"
```

Comment:
Would prefer a more descriptive name, like `InspectAnnotationValueDisabled` or something.

Reply:
yes I can do that, kept it anonymous to annotation in case it might be used in other context as well

Reply:
In general I think you only want things to share the same definition when there's a causal relationship between them (even if that means repeating yourself sometimes).

Reply:
Agreed, done!